testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/ModuleTestCase.java

```diff
@@ -186,6 +186,20 @@ public void addRemoveModuleMetaInf() throws Exception {
         testModuleAndSlotExist(false, slot, getModulePath());
     }
 
+    @Test
+    public void addModuleWithWhitespacesInDependencies() throws Exception {
+        // should trim the surrounding whitespaces
+        try {
+            cli.sendLine("module add --name=" + MODULE_NAME
+                    + " --resources=" + jarFile.getAbsolutePath()
+                    + " --dependencies=[ org.jboss.logging ]"
```

Comment:
using the '[' and ']' characters for dependencies seems not right. Same for export-dependencies.